contrib/packs/actions/create.meta.yaml

```diff
@@ -0,0 +1,41 @@
+---
+  name: "create"
+  runner_type: "action-chain"
+  description: "Creates a new pack"
+  enabled: true
+  entry_point: "workflows/create.yaml"
+  parameters:
+    name:
+      type: "string"
+      required: true
+      description: "Name of the pack to create"
+    description:
```

Comment:
Probably. This spec is a conversation piece more than proposal, really, I don't have strong feelings on pack.yml content.